### ISSUE TITLE
A11y_AzureToolsStorageExplorer_ToggleExplorerPaneTreeView_Keyboard: Keyboard users not able to Open Context menus using shortcut key "Shift+F10".

### ISSUE THREAD

**Issue by vinodbk197** [microsoft/AzureStorageExplorer#2613]
"[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier."
​

**GitHubTags:** #A11y_AzureToolsStorageExplorer; #A11YTCS; #DesktopApp; #Win32; #Benchmark; #A11ySev2; #MAS2.1.1; #Keyboard;

**Environment Details:**
Application Name: Azure Tools Storage Explorer
Application Version: 1.12.0
Windows10

**Repro Steps:**
1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Traverse through the treeview opened on left of screen for whatever tasks or subscription selected.
5. Navigate to controls present under the "Blob Containers"
6. Check context menu opens after pressing "Shift+F10" key.

**Actual:**
Keyboard users not able to Open Context menus using shortcut key "Shift+F10".

**Expected**:
By Pressing "Shift+F10" key should open a context menu for the controls present under Explorer Pane Tree View section.

**User Impact:**
Keyboard users and Screen reader users are unable to reliably complete actions that require accessing the context menu, because it is not obvious how to open it from the keyboard.

**Recommendation:**
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**MAS Reference:**
**MAS2.1.1:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

**Comment by vinodbk197** [comment]
We have uploaded repro video of this bug in [OneDrive](https://microsoft-my.sharepoint.com/personal/v-subha_microsoft_com/_layouts/15/onedrive.aspx?id=%2Fpersonal%2Fv%2Dsubha%5Fmicrosoft%5Fcom%2FDocuments%2FMicrosoft%20Azure%20Tools%20Storage%20Explorer%20Attachments&originalPath=aHR0cHM6Ly9taWNyb3NvZnQtbXkuc2hhcmVwb2ludC5jb20vOmY6L3Avdi1zdWJoYS9FdnhIZGcxemtOOVBrbkYxdU94bk9pUUJlanB4S1lnOW4yMUR5ZGdRX29tMGV3P3J0aW1lPWJNTVh6dTJ1MTBn) link with name "2613_A11y_AzureToolsStorageExplorer_ToggleExplorerPaneTreeView_Keyboard" please refer the Screenshot for reference.
Please let us know if there is any problem in accessing the link

**Comment by JasonYeMSFT** [comment]
I don't have access to the OneDrive you are sharing. Plus I cannot reproduce the same issue on my machine. Please attach the reproduce files here so we can access them.

**Comment by vinodbk197** [comment]
Verified this issue again, The issue is not reproduces ,Hence closing this bug.

**Comment by Suraj1200499** [comment]
@JasonYeMSFT I have Shared a One drive link with you and Matthew. Please let us know if you are able to access the link.

**Comment by JasonYeMSFT** [comment]
I guess I know what happened for you to think there is a bug. If you click on the blob container node (the one you clicked in the repro video), we automatically open its explorer and transfer the focus to the 1st toolbar button in the explorer. If you then press Shift + F10, it won't open the context menu because the element in focus (toolbar button) doesn't have one. Since this would only happen when you use mouse, we don't think this is an accessibility issue.
